dorm/dorm/Presenter/Sidebar/Sidebar.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Sidebar.swift
+//  dorm
+//
+//  Created by JongHo Park on 2022/07/20.
+//
+
+import UIKit
+
+final class Sidebar: UIView {
```

Comment:
```suggestion
final class SidebarView: UIView {
```

UIKit에서는 뒤에 View를 붙여주는 것이 일반적인 명명이랍니다 ~

Reply:
감사합니다! 바로 수정하겠습니다

Reply:
77c4dc61d6f603aa9de00456b2274cdf43f99689 에서 수정했습니다!